INSTALL.md

```diff
@@ -101,6 +115,14 @@ The [Releases](https://github.com/neovim/neovim/releases) page provides an [AppI
     chmod u+x nvim.appimage
     ./nvim.appimage
 
+To expose nvim globally:
+
+    mkdir -p $HOME/.local/bin
+    mv nvim.appimage $HOME/.local/bin
+    ln -s $HOME/.local/bin/nvim.appimage $HOME/.local/bin/nvim
+    nvim
```

Comment:
Not sure why we're telling people how to use `mv` and `ln -s`. The docs need to stay focused.

The `/opt/` suggestion for the other case in this PR is ok I guess, it saves some decisions for the user. But this section needs to be deleted, also because it distracts from the "squashfs" followup below.

Reply:
I saw `ln` being used when I searched what the typical ways of installing programs without a package manager were. If it should be used, then I think it's useful to include the commands, since the `nvim` command should be exposed globally (for most people.) And it's nice to list a working command for people who want to get up and running.

Reply:
> Could probably just remove the ln command, as using mv to remove the .appimage would be fine

yeah that seems reasonable and avoids some steps.